panel/io/state.py

```diff
@@ -333,13 +335,18 @@ async def _scheduled_cb(self, name: str) -> None:
     # Public Methods
     #----------------------------------------------------------------
 
-    def as_cached(self, key: str, fn: Callable[[], None], ttl: int = None, **kwargs) -> None:
+    def as_cached(self, key: str, fn: Callable[[], T], ttl: int = None, **kwargs) -> T:
```

Comment:
```suggestion
    def as_cached(self, key: str, fn: Callable[[], T], ttl: int = None, **kwargs) -> T:
```

Can you explain the use of `T`? Shouldn't it simply be `Any`?

Reply:
Nope :-) I want the editor to understand that `as_cached` return the same value (`T`) as the callable `fn` - not Any as I then will not be able to get any help from my editor from that point.

Reply:
Ah got it.